### ISSUE TITLE
update is-glob to 4.0.0

### ISSUE THREAD

**Issue by pravi** [gulpjs/glob-parent#16]
One test is failing though, see https://github.com/micromatch/glob-base/pull/5 for a discussion

```
 1) glob-parent should strip glob magic to return parent path:

      AssertionError: qmarks must be escaped
      + expected - actual

      -path/?
      +path
      
      at Context.<anonymous> (/home/pravi/forge/debian/git/pkg-javascript/node-glob-parent/test.js:48:12)
      at callFn (/usr/lib/nodejs/mocha/lib/runnable.js:223:21)
      at Test.Runnable.run (/usr/lib/nodejs/mocha/lib/runnable.js:216:7)
      at Runner.runTest (/usr/lib/nodejs/mocha/lib/runner.js:373:10)
      at /usr/lib/nodejs/mocha/lib/runner.js:451:12
      at next (/usr/lib/nodejs/mocha/lib/runner.js:298:14)
      at /usr/lib/nodejs/mocha/lib/runner.js:308:7
      at next (/usr/lib/nodejs/mocha/lib/runner.js:246:23)
      at Immediate.<anonymous> (/usr/lib/nodejs/mocha/lib/runner.js:275:5)
      at runCallback (timers.js:672:20)
      at tryOnImmediate (timers.js:645:5)
      at processImmediate [as _immediateCallback] (timers.js:617:5)
  
```

**Comment by mrmlnc** [comment]
@es128, ping ❤️

**Comment by es128** [comment]
@mrmlnc what are you asking me to do? Has someone worked out a solution?

**Comment by phated** [comment]
@evilebottnawi yes please!

**Comment by phated** [comment]
Closed by https://github.com/gulpjs/glob-parent/commit/988ec773f6b4002f7c60c3fd2e88ccaf25f0ee07 - being released in 4.0.0
